diff --git a/php7/memcache_session.c b/php7/memcache_session.c
index dd2434a..45ba97d 100644
--- a/php7/memcache_session.c
+++ b/php7/memcache_session.c
@@ -289,7 +289,7 @@ PS_READ_FUNC(memcache)
 			ZVAL_NULL(&addresult);
 
 			/* third request fetches the data, data is only valid if either of the lock requests succeeded */
-			ZVAL_NULL(&dataresult);
+			ZVAL_EMPTY_STRING(&dataresult);
 
 			/* create requests */
 			if (php_mmc_session_read_request(pool, &zkey, lockparam, &addresult, dataparam, &lockrequest, &addrequest, &datarequest) != MMC_OK) {
